src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -157,6 +158,57 @@ export const PlatformSwitch: FC<PlatformSwitchProps> = ({
     />
   ));
 
+  if (isMobile) {
+    const SelectedIcon = PlatformIcons[selected || 'Builder'];
+    return (
+      <Box sx={{ position: 'relative' }}>
+        <SpeedDial
+          ariaLabel='SpeedDial basic example'
+          sx={{
+            position: 'absolute',
+            '& button': {
+              width: '40px',
+              height: '40px',
+            },
+          }}
+          icon={selected && <SelectedIcon />}
+          direction={'down'}
+        >
+          {Object.values(Platform).map((platform, index) => {
+            const Icon = PlatformIcons[platform];
+            const isSelectedPlatform = platform === selected;
+            const platformProps = platformsProps?.[platform];
+            const iconProps = {
+              size,
+              secondaryColor: isSelectedPlatform ? color : accentColor,
+              primaryColor: isSelectedPlatform ? accentColor : color,
+              fill: isSelectedPlatform ? color : accentColor,
```

Comment:
I would prefer to have the colors inverted, so the selected platform is on a blue background and the non selected platforms are on a white background:
<img width="80" alt="Screenshot 2024-01-05 at 15 36 04" src="https://github.com/graasp/graasp-ui/assets/39373170/2deae7c2-807d-42a5-b271-e861ef9ba98f">

Reply:
Actually I got confused by the fact that the "right" color scheme is in the "dark" story. But when viewing it in "light" mode it looks right in the expanded version but not the mobile one... So we should either remove the light version story, or use a different background color to match the effect that the expanded version has. Not sure this is clear ...